### PR TITLE
Tyler Gilchrist

### PR DESCRIPTION
**Pull request by Tyler668**

### CODE REVIEW

**Comment by Tyler668** [comment]
One off of 10 commits... spare me Sydney

File: README.md [hunk 1]
```diff
@@ -30,32 +30,53 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+.forEach will go through an array and execute a provided function once for each item, changing the values in the array. On the other hand, .map will go through the items but create a new array of applied changes, leaving the original intact.
```
Comment (srsimps19) [inline README.md:33]: Great answer!!

File: challenges/classes.js [hunk 2]
```diff
@@ -1,7 +1,54 @@
...
+        return (this.length * this.width * this.height);
+    }
+
+    surfaceArea() {
+        return (2 * (this.length * this.width + this.length * this.height + this.width * this.height));
+    }
+
+};
+
+const cubey2 = {
```
Comment (srsimps19) [inline challenges/classes.js:20]: forever loving your naming conventions lol

File: challenges/functions.js [hunk 3]
```diff
@@ -4,29 +4,47 @@
...
 
 
 // ==== Closures ==== 
 
 // Explain in your own words why `nestedfunction()` can access the variable `internal`.
 
 // Explanation: 
-
+// By instantiating nestedFunction inside of myFunction, we have extended its scope to include variables 
+// that are within the parent function that it normally would not have access to.
```
Comment (srsimps19) [inline challenges/functions.js:47]: Great answer!

**Review by srsimps19 — APPROVED** [review]
Keep up the good work Tyler!

**Pull request merged**